packages/ragbits-core/docs/how-to/hybrid-search.md

```diff
@@ -0,0 +1,169 @@
+# How to Implement Hybrid Search with Dense and Sparse Embeddings
```

Comment:
We already have a how-to on Hybrid Search, and this is the 3rd (!) document on this that you add in this PR.

Besides, we keep our documentation in the top-level `docs/` directory, not inside individual packages.